Author: Agustin Martin Domingo <agmartin@debian.org>
Description: Adding info to the header about Debian ispell.el modifications

Index: dictionaries-common/support/emacsen/ispell.el
===================================================================
--- dictionaries-common.orig/support/emacsen/ispell.el	2014-10-14 18:27:30.927500926 +0200
+++ dictionaries-common/support/emacsen/ispell.el	2014-10-14 18:27:30.919500943 +0200
@@ -26,6 +26,17 @@
 ;; You should have received a copy of the GNU General Public License
 ;; along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.
 
+;; --------------------------------------------------------------------
+;; This file has been modified by Rafael Laboissiere <rafael@debian.org>
+;; and Agustin Martin <agmartin@debian.org> for the Debian GNU/Linux
+;; system to meet the requirements of the Debian Spelling Policy.
+;;
+;; Amongst other minor changes, ispell-dictionary-alist is set to
+;; debian-ispell-dictionary-alist if present, containing entries for the
+;; really installed and registered dictionaries, and ispell-dictionary is
+;; set to the debian debconf selected value if possible
+;; --------------------------------------------------------------------
+
 ;; Note: version numbers and time stamp are not updated
 ;;   when this file is edited for release with GNU Emacs.
 
